### DISCUSSION TITLE
Slider event on release

### DISCUSSION THREAD

**Discussion by aljosahafner** [zauberzeug/nicegui#1902] [Q&A] [answered]
### Question

Hey,

The user wants to control a given parameter change with a slider, but there is a short (but noticeable) calculation behind changing the value. Of course one way is to use the throttle method, but it's not the most convenient. Now the question is whether there is a specific event that is triggered when the slider is released (at the end of dragging/setting)?

Something along these lines:
```py
ui.slider(min=0, max=10, step=0.1, value=5).props('label-always').on('dragend', lambda e: execute_some_code(e.args))
```

Thanks in advance!

**Comment by falkoschindler** [comment, marked as answer]
Yes, there is:

```py
ui.slider(min=0, max=10, step=0.1, value=5).props('label-always').on('change', lambda e: ui.notify(e.args))
```

The [Quasar documentation](https://quasar.dev/vue-components/slider#qslider-api) lists all available events on the "Events" tab.

**Comment by aljosahafner** [comment]
Ooooo! Wow, thanks a lot for this wonderful support!
